### PR TITLE
fix: Selectability of Android `TextBoxView`

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/11665

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Fix `TextBoxView` selectability when not `ReadOnly`

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs [hunk 1]
```diff
@@ -421,24 +423,24 @@ private void UpdateTextBoxViewReadOnly()
...
 			}
+			else if (_listener is not null)
+			{
+				_textBoxView.KeyListener = _listener;
+			}
+
+			_textBoxView.Focusable = IsTabStop;
+			_textBoxView.FocusableInTouchMode = IsTabStop;
+			_textBoxView.Clickable = IsTabStop;
+			_textBoxView.LongClickable = IsTabStop;
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs:442]: This is the opposite of the original code, is this the fix?

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs:442]: Yes, actually the read-only part is handled by the KeyListener. The IsTabStop should decide the interactivity, I have tested all 4  different configurations and it behaves as in WinUI (at least on my side)

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs:442]: Will verify this

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs:442]: Finally found the solution!

File: src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs [hunk 2]
```diff
@@ -236,8 +237,7 @@ protected void SetInputScope(InputTypes types)
 		{
 			if (_textBoxView != null)
 			{
-				_textBoxView.InputType = types;
-				_textBoxView.SetRawInputType(types);
+				_textBoxView.SetInputTypes(types, types);
```
[the remark this answers is not in the document]
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs:240]: Oh I see now it uses SetRawInputType internally. I confused the new method you wrote with `InputTypes` property.

**Review by Youssef1313 — APPROVED** [review]

**Pull request merged**